### DISCUSSION TITLE
A good formatting VS Code extension for .ejs files?

### DISCUSSION THREAD

**Discussion by astrowonk** [quarto-dev/quarto-cli#8022] [Q&A]
### Description

I have one custom .ejs file for my Projects section of my web site. When I need to edit it, it's nice to have formatting and color syntax highlighting in VS Code.

But "[EJS language support](https://marketplace.visualstudio.com/items?itemName=DigitalBrainstem.javascript-ejs-support)" I'm pretty sure just malformed and broke one of my custom .ejs files, so I had to revert via git. I'm now trying [EJS Beautify](https://marketplace.visualstudio.com/items?itemName=j69.ejs-beautify) which seems to work ok...

But Quarto Devs, you must be in and out of EJS files all the time. What sort of tools in VS Code do you use to help you edit and format them?

**Comment by mcanouil** [comment]
I suggest to look on the marketplace: https://marketplace.visualstudio.com/search?term=ejs&target=VSCode&category=All%20categories&sortBy=Relevance

Note that EJS is really a very minor part of the codebase.
